docs/api/class-reference.md

```diff
@@ -186,3 +186,8 @@ This object represents a generic Creative. It's used as a parent object for more
 - `type: String|null` The type of executable resource provided
 - `parameters: String|null`  String intended for bootstrapping the verification code and providing metadata about the current impression
 - `trackingEvents: Object`
+
+## BlockedAdCategories<a name="blocked-ad-categories"></a>
+
+- `attributes: Object`
```

Comment:
Our approach to parsing in the vast-client is to put each attribute as a new property of this object. So in this case, it would be: `authority: String` instead of `attributes: Object`

Reply:
Also this should be mentioned in `Ad` documentation, together with all the new fields added :)